### PR TITLE
Implemented time out report to log cluster status and condition infor…

### PR DESCRIPTION
**Pull request by lscalabrini01**
- Changed provisioning tests to use TimeoutRKEReport and TimeoutClusterReport

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -14,6 +14,7 @@ replace (
 
 	github.com/rancher/rancher/pkg/apis => ./pkg/apis
 	github.com/rancher/rancher/pkg/client => ./pkg/client
+	github.com/rancher/shepherd => github.com/lscalabrini01/shepherd v0.0.0-20240705200636-a86b49c33452
```
Comment (floatingman) [inline go.mod:17]: Be sure to change this before merging

**Review by floatingman — CHANGES_REQUESTED** [review]
Looks good. Do you have test runs in Jenkins? If so, can you post it in the test-automation channel?

File: go.mod (hunk 1 above)
Comment (lscalabrini01) [reply go.mod:17]: Yes, for sure

**Review by floatingman — APPROVED** [review]
LGTM

**Pull request closed without merging**